processor/batchprocessor/metrics.go

```diff
@@ -56,8 +57,18 @@ func newBatchProcessorTelemetry(set processor.CreateSettings, currentMetadataCar
 }
 
 func (bpt *batchProcessorTelemetry) createOtelMetrics(set component.TelemetrySettings, currentMetadataCardinality func() int) error {
-	var errors, err error
-	meter := metadata.Meter(set)
+	var (
+		errors, err error
+		meter       metric.Meter
+	)
+
+	// BatchProcessor metrics are not subject to the same level of filtering as other components.
```

Comment:
Isn't this true for all components? If the default is normal, aren't all metrics (other than instrumentation library metrics) emitted?

Reply:
> Isn't this true for all components?

Yes, it's true for all components. With this comment, I wanted to say that the batch processor is the only component in core emitting metrics (with the normal level). It's probably confusing. Let me update it

> If the default is normal, aren't all metrics (other than instrumentation library metrics) emitted?

Right